documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -6,15 +6,15 @@
 [id='assembly-upgrade-{context}']
 = Upgrading Strimzi
 
-Strimzi can be upgraded with no cluster downtime.
+Strimzi can be upgraded without causing any downtime for consumers and producers. 
```

Comment:
I'm not sure how to sort out `can be upgraded without causing any downtime` with `availability might be affected`. I think this would need to better explain what do we mean with `cluster availability` and how does it differ from consumer / producer downtime.

I also wonder if we should add something like _without causing any downtime for consumers and producers using highly available topics_ or something.

Reply:
Great comment, thank you. I added explanations of downtime and cluster availability. Please can you review.